StressTest.ps1

```diff
@@ -0,0 +1,24 @@
+﻿[CmdletBinding()]
+param(
+    [string]
+    $BinSkimFolder = "bld\bin\x64_Release\netcoreapp3.1",
```

Comment:
>$BinSkimFolder [](http://example.com/codeflow?start=4&length=14)

Can we put this test file in a better place and use a relative path to get to this folder?